.changeset/fair-otters-shop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@imgproxy/imgproxy-js-core": major
+---
+
+BREAKING CHANGE: rename `ajust` option to `adjust` and its shorthand counterpart `aj` to `a`
```

Comment:
De-facto, it's not a breaking change but a fix

Reply:
Hmm, right, if it never worked before this is a fix indeed. I'll change it to `minor`